### PR TITLE
SQL information schema documentation

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-iox/reference/sql/information-schema.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+menu:
+  influxdb_cloud_iox:
+    parent: SQL reference
+weight: 210
+---
+
+The underlying query engine for the InfluxDB SQL implementation,
+[DataFusion](https://arrow.apache.org/datafusion/index.html), provides commands
+that return metadata related to your data schema.
+To access this information, use the the `SHOW TABLES`, `SHOW COLUMNS`, and
```
Comment (jstirnaman) [inline content/influxdb/cloud-iox/reference/sql/information-schema.md:14]:
```suggestion
To access this information, use the `SHOW TABLES`, `SHOW COLUMNS`, and
```

File: content/influxdb/cloud-iox/reference/sql/information-schema.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+`SHOW ALL` commands or query views of in [ISO](https://www.iso.org/) SQL
```
Comment (jstirnaman) [inline content/influxdb/cloud-iox/reference/sql/information-schema.md:15]:
```
`SHOW ALL` commands or query views of in [ISO](https://www.iso.org/) SQL
```

I'm not sure what this is supposed be, but seems like a typo?

File: content/influxdb/cloud-iox/reference/sql/information-schema.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+`information_schema` schema.
+
+In the context of InfluxDB a [measurement](/influxdb/cloud-iox/reference/glossary/#measurement)
```
Comment (jstirnaman) [inline content/influxdb/cloud-iox/reference/sql/information-schema.md:18]:
```suggestion
In the context of InfluxDB, a [measurement](/influxdb/cloud-iox/reference/glossary/#measurement)
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
A few small things.

File: content/influxdb/cloud-iox/reference/sql/information-schema.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/cloud-iox/reference/sql/information-schema.md:15]:
```suggestion
`SHOW ALL` commands or query views in the [ISO](https://www.iso.org/) SQL
```

File: content/influxdb/cloud-iox/reference/sql/information-schema.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
+---
+title: Information schema
+description: > 
+    ...
```
Comment (sanderson) [inline content/influxdb/cloud-iox/reference/sql/information-schema.md:4]:
```suggestion
  The `SHOW TABLES`, `SHOW COLUMNS`, and `SHOW ALL` commands return metadata related to
  your data schema.
```

**Review by jstirnaman — APPROVED** [review]
🎸

**Pull request merged**